core/controlplane/config/templates/cloud-config-controller

```diff
@@ -814,11 +814,11 @@ write_files:
       kubectl create secret generic kiam-server-tls -n kube-system \
```

Comment:
We dump a yaml of the secret containing `namespace: kube-sytem` by specifying `-n kub-system` here.
So, `kubectl apply -f -` should create the secret inside the kube-system namespace, even though we don't have `-n kube-system` for `kubectl apply`.
But maybe I'm missing something?

Also see https://github.com/kubernetes-incubator/kube-aws/issues/1163#issuecomment-369452853